ccmlib/repository.py

```diff
@@ -113,10 +113,32 @@ def clone_development(git_repo, version, verbose=False):
                 else:
                     subprocess.call(['git', 'clone', local_git_cache, target_dir], cwd=__get_dir(), stdout=lf, stderr=lf)
 
+                # determine if the request is for a branch
+                is_branch = False
+                try:
+                    branch_listing = subprocess.check_output(['git', 'branch', '--all'], cwd=target_dir)
+                    branch_listing = branch_listing.replace('remotes/origin/', '')
+                    for branch in branch_listing.splitlines(True):
+                        if git_branch == branch.strip():
```

Comment:
It might be better to do `if git_branch in branch.strip():` here, just in case, no?